lib/components/molecules/Table/Table.js

```diff
@@ -0,0 +1,69 @@
+// @flow
+
+import React from 'react';
+import styled from 'styled-components';
+
+import styles from './Table.style';
+import type { Props, HeadProps } from './types';
+
+const Head = ({ children }: HeadProps) => <th>{children}</th>;
+
+const Cell = ({ children, item }: Props) => {
```

Comment:
This component should need a separate flow definition.

Reply:
done